pkg/containerruntime/crio/supported_versions.go

```diff
@@ -22,6 +22,8 @@ func GetOfficiallySupportedVersions(kubernetesVersion string) ([]string, error)
 		return []string{"1.9"}, nil
 	case "1.10":
 		return []string{"1.10"}, nil
+	case "1.11":
+		return []string{"1.10"}, nil
```

Comment:
Looking at the fedora packages, they have a `1.11` package.
But not in the ubuntu repos :/